include/rocksdb/table.h

```diff
@@ -86,6 +86,9 @@ struct BlockBasedTableOptions {
     // The hash index, if enabled, will do the hash lookup when
     // `Options.prefix_extractor` is provided.
     kHashSearch,
+
+    // A two-level index implementation. Both levels are binary search indexes.
+    kTwoLevelIndexSearch,
```

Comment:
Or we can simply add comment here saying the feature is experimental and not ready for use.